sharpy/solvers/nonlineardynamicmultibody.py

```diff
@@ -178,6 +215,20 @@ def extract_resultants(self):
         pass
 
     def compute_forces_constraints(self, MB_beam, MB_tstep, ts, dt, Lambda, Lambda_dot):
+        """
+        This function computes the forces generated at Lagrange Constraints
+
+        Args:
+            MB_beam (list(``Beam``)): each entry represents a body
+            MB_tstep (list(``StructTimeStepInfo``)): each entry represents a body
+            ts (int): Time step number
+            dt(int): time step
+            Lambda (np.ndarray): Lagrange Multipliers array
+            Lambda_dot (np.ndarray): Time derivarive of ``Lambda``
+
+        Notes:
```

Comment:
```suggestion
        Warning:
```
Using `Warning:` puts it into an orange box in the docs. `Note:` puts it in a blue box but I believe `Notes:` is just a normal heading.